AdfDataflowFilePrettifier/Prettifier.cs

```diff
@@ -42,18 +43,14 @@ public static bool VerifyUgliness(string existingFileContents)
             return !FileContainsDataFlow(existingFileContents) || !FileIsCurrentlyPretty(existingFileContents);
         }
 
-
-
-
-
         private static bool FileContainsDataFlow(string fileContents)
         {
             return fileContents.Contains("\"type\": \"MappingDataFlow\",");
         }
 
         private static bool FileIsCurrentlyPretty(string existingFileContents)
         {
-            return existingFileContents.Contains(InnerMagicIndicatorCharacter) || existingFileContents.Contains(OuterMagicIndicatorCharacter) || existingFileContents.Contains(CRMagicIndicatorCharacter);
+            return existingFileContents.Contains(BlockStartString) || existingFileContents.Contains(BlockEndString);

```

Comment:
Can you add something here that also checks for stray whitespace in the script definition?
One quick option is probably to just validate it as a JSON string?
https://stackoverflow.com/a/14977915/1662268

Reply:
Although ... feel free to declare that that's outside the scope of this change, and leave me to do it if I want it.

Reply:
I haven't got time to change this right now, but happy to do so when as part of v2 markups